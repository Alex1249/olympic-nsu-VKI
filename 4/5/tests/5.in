42 20
31 977 364 342 776 840 -733 313 -518 974 -38 -698 -954 240 364 -920 529 -801 -989 -881 -126 545 -814 -821 381 -906 -502 960 264 -642 24 -931 -54 -156 837 -700 479 695 -616 -490 977 789
1 1017
1 697
0 767
0 85
0 2
1 1204
0 20
0 657
1 1053
0 302
1 1036
1 105
1 211
0 1104
1 916
0 438
1 406
0 1031
1 1213
0 792
