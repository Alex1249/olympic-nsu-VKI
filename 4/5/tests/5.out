31 977 364 4194646 776 840 -733 313 -518 974 -38 -698 -442 240 364 -920 529 -801 -989 -881 -16510 609 -814 -822 381 -906 -502 960 2312 -642 24 -931 -16777266 -156 837 -700 479 2743 -616 -490 977 789 
